test/fixtures/rules/std_rule_cases/L046.yml

```diff
@@ -31,12 +36,21 @@ test_fail_templated_segment_contains_leading_literal:
     SELECT user_id
     FROM
         `{{"gcp_project"}}.{{"dataset"}}.campaign_performance`
+  fix_str: |
+    SELECT user_id
+    FROM
+        `{{ "gcp_project" }}.{{ "dataset" }}.campaign_performance`
   configs:
     core:
       dialect: bigquery
 
 test_fail_segment_contains_multiple_templated_slices_last_one_bad:
   fail_str: CREATE TABLE `{{ "project" }}.{{ "dataset" }}.{{"table"}}`
+  fix_str: CREATE TABLE `{{ "project" }}.{{ "dataset" }}.{{ "table" }}`
   configs:
     core:
       dialect: bigquery
+
+test_fail_jinja_tags_no_space_no_content:
+  fail_str: SELECT {{-""-}}1
```

Comment:
Worth adding more tricky tests where the template output crosses segment boundaries? E.g.

```
SELECT a{{-"1 + b"}}
```

```
SELECT a{{-"1 + b"-}}2
```

Reply:
ooooh good idea